dotCMS/src/main/java/com/dotcms/rendering/velocity/services/StringifyObject.java

```diff
@@ -0,0 +1,104 @@
+package com.dotcms.rendering.velocity.services;
+
+import com.dotcms.repackage.com.ibm.icu.text.SimpleDateFormat;
+
+import java.io.StringWriter;
+import java.util.Collection;
+import java.util.Date;
+import java.util.Iterator;
+
+final class StringifyObject {
+    final String stringified;
+
+    public StringifyObject(final Object o) {
+
+        if(o instanceof String[]) {
+            this.stringified = stringifyObject((String[]) o);
+        }
+        else if(o instanceof Collection) {
+            this.stringified = stringifyObject((Collection) o);
+        }
+        else if(o instanceof Boolean) {
+            
+            this.stringified = stringifyObject((Boolean) o);
+        }
+        else if(o instanceof Date) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)